__tests__/component/form/CreateTaskForm.test.tsx

```diff
@@ -136,34 +136,3 @@ test("onCancel is invoked", async () => {
 
     await waitFor(() => expect(onCancelSpy.callCount).toEqual(1))
 })
-
```

Comment:
Let's remove the line 16
```
import _ from "loadsh
```
so that the linter won't complain. You can also remove the `disable-eslint` from the line above that one.

Reply:
I'm maybe getting this wrong but removing this import statement breaks the functionality of missing required field-test.